modules/java_api/samples/benchmark_app/Main.java

```diff
@@ -91,7 +94,7 @@ static Blob blobRandomByte(TensorDesc tDesc) {
         }
 
         byte[] buff = new byte[size];
-        Random rand = new Random();
+        SecureRandom rand = new SecureRandom();
```

Comment:
Why do we need `SecureRandom` here?

Reply:
Static code analyzer stays that Random is not secure :)